calicoctl/commands/node/status.go

```diff
@@ -160,5 +193,9 @@ func drawTable(birdOut string) {
 	for _, v := range data {
 		table.Append(v)
 	}
+	if len(data) == 0 {
+		return errors.New("No peers found")
```

Comment:
Instead of returning an error here, why don't we just print out "No IPv%s BGP peers found"?  

This function doesn't really need to return errors.